### PR TITLE
Add host --headers flag to fortio command for perf test

### PR DESCRIPTION
**Pull request by carolynhu**
https://github.com/istio/istio/issues/21687

In perf test we are passing the headers like: `--headers=foo:bar,foo1:bar1,foo2:bar2`, and in fortio, it will be like: `-H=foo:bar -H=foo1:bar1  -H=foo2:bar2`

### CODE REVIEW

File: perf/benchmark/README.md [hunk 1]
```diff
@@ -113,6 +113,7 @@ Required fields to specified via CLI or config file:
 ```bash
 optional arguments:
   -h, --help            show this help message and exit
+  --header HEADER       header:value, can be specified multiple times to add headers \(including Host:\)
```
Comment (mandarjog) [inline perf/benchmark/README.md]: If this is multi valued then it should be 
—headers

Comment (carolynhu) [reply perf/benchmark/README.md]: yeah, you are right.

**Comment by carolynhu** [comment]
> what is this for? Isn't fortio already adding host? Or is it for ingress

Yes, fortio natively support -H flag, but we haven't used it yet in our perf test. Yeah, ingress needs it.

File: perf/benchmark/README.md (hunk 1 above)
Comment (howardjohn) [reply perf/benchmark/README.md]: for what its worth the flag is `--header` in curl

File: perf/benchmark/runner/runner.py [hunk 2]
```diff
@@ -183,8 +185,9 @@ def run(self, conn, qps, size, duration):
             cacert_arg = "-cacert {cacert_path}".format(cacert_path=self.cacert)
 
         fortio_cmd = (
-            "fortio load -c {conn} -qps {qps} -t {duration}s -a -r {r} {cacert_arg} {grpc} -httpbufferkb=128 " +
+            "fortio load -H={headers} -c {conn} -qps {qps} -t {duration}s -a -r {r} {cacert_arg} {grpc} -httpbufferkb=128 " +
```
Comment (carolynhu) [inline perf/benchmark/runner/runner.py]: @howardjohn I think `--header` or `--headers` just the flag that we want to pass values to. For fortio, it actually use `-H` underneath. So, what do you think?

File: perf/benchmark/README.md (hunk 1 above)
Comment (mandarjog) [reply perf/benchmark/README.md]: Either use —header but you can supply the option multiple times, or —headers with all header as a single argument.
This pr was doing a combination

**Pull request merged**